### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by shiyuanh** [chaoyuaw/pytorch-coviar#56]
Are the training parameters specified in the GETTING_STARTED the exact parameters that produce the expected results? (They look different from the paper). I tried the same setting given in the template on UCF101 training split 1, but the test accuracy on testing split 1 is very low. (<0.05)

**Comment by chaoyuaw** [comment]
Thanks for the questions. Yes, as explained in
https://github.com/chaoyuaw/pytorch-coviar/blob/master/GETTING_STARTED.md#training

"The hyperparameters here are slightly different from those used in the original paper, because the pre-trained weights are different. The original paper uses ResNet (pre-activation) pre-trained by MXNet, here we use ResNet (non-pre-activation) pre-trained by PyTorch. They offer similar results."

The commands in GETTING_STARTED should reproduce the results in paper. I'm not entirely sure what caused for the low accuracy, but maybe it worths trying visualize the "input" of the network (e.g., transform each batch back to [0, 255] unit8 arrays and store back as jpeg images) to see if they makes sense.

**Comment by chaoyuaw** [comment]
Also, were you able to re-encode the videos successfully (https://github.com/chaoyuaw/pytorch-coviar/blob/master/GETTING_STARTED.md#datasets)?

**Comment by shiyuanh** [comment]
Thanks for your responses. Yes I was able to re-encode the videos. I have fixed my low accuracy problem --- it might be a problem with my pytorch version.
